server/src/main/java/io/crate/metadata/settings/CoordinatorSessionSettings.java

```diff
@@ -42,6 +44,7 @@ public class CoordinatorSessionSettings extends SessionSettings {
     private User sessionUser;
     private Set<Class<? extends Rule<?>>> excludedOptimizerRules;
     private String applicationName;
+    private String dateStyle;
```

Comment:
I think it needs to be moved to `SessionSettings` and only the setter should be left here, similar to other params like `searchPath`.

Reply:
As discussed, I think it's fine to keep it as is, and not stream it with `SessionSettings`.
@mfussenegger do you agree?

Reply:
This is fine as it is. We don't want to stream the value across nodes as it adds overhead.